tests/README.md

```diff
@@ -15,22 +15,51 @@ Developers can disable the tests by adding `[skip ci]` in the last commit messag
 
 ### Build settings
 
-Use "Other" language and the following build commands:
+Select the "Other" language.
```

Comment:
From time to time somebody is asking why don't we use the "Go" setting. Maybe add a paragraph like following:

"We don't use "Go" language setting, because rkt is not a typical go project (building it with a `go get` won't get you too far). Also, the "Go" setting is creating a proper GOPATHy directory structure with some symlinks on top, which rkt does not need at all and some go tools we use do not like the symlinks in GOPATH at all."